reana_client/cli/workflow.py

```diff
@@ -455,6 +454,61 @@ def add_verbose_data_from_response(response, verbose_headers,
                 err=True)
 
 
+@click.command(
+    'du',
+    help='Get disk usage of a workflow.')
+@click.option(
+    '-w',
+    '--workflow',
+    default=os.environ.get('REANA_WORKON', None),
+    callback=workflow_uuid_or_name,
+    help='Name or UUID of the workflow to display the disk usage. '
+         'Overrides value of REANA_WORKON environment variable.')
+@add_access_token_options
+@click.option(
+    '-s',
+    '--summarize',
+    count=True,
+    help='Display total.')
+@click.pass_context
+def workflow_disk_usage(ctx, workflow, access_token, summarize):
+    """Get disk usage of a workflow."""
+    logging.debug('command: {}'.format(ctx.command_path.replace(" ", ".")))
+    for p in ctx.params:
+        logging.debug('{param}: {value}'.format(param=p, value=ctx.params[p]))
+
+    if not access_token:
+        click.echo(
+            click.style(ERROR_MESSAGES['missing_access_token'],
+                        fg='red'), err=True)
+        sys.exit(1)
+
+    if workflow:
+        try:
+            parameters = {'summarize': summarize}
+            response = get_workflow_disk_usage(workflow,
+                                               parameters,
+                                               access_token)
+            headers = ['size', 'name']
+            data = []
+            if summarize:
+                click_table_printer(headers, [],
+                                    [response['disk_usage_info'][-1]['size'],
+                                     response['disk_usage_info'][-1]['name']])
```

Comment:
Works nicely; one remark on the output:

```console
$ reana-client du -w helloworld.serial          
SIZE   NAME                  
4.0K   /results/greetings.txt
4.0K   /results              
4.0K   /data/names.txt       
4.0K   /data                 
4.0K   /code/helloworld.py   
4.0K   /code                 
12K                          
```

It would be good to display `./results/greetings.txt' with the leading `./` so that directories wouldn't appear to be absolute but rather relative paths to the workflow workspace.